package.json

```diff
@@ -1,10 +1,10 @@
 {
-	"name": "Softcore",
-	"version": "2.1.2",
-	"main": "src/mod.js",
+	"name": "softcore",
+	"version": "3.0.0",
+	"main": "src/Softcore.js",
 	"license": "MIT",
 	"author": "ODT",
-	"sptVersion": "~3.9",
+	"sptVersion": "3.10.3",
```

Comment:
Changing this to `"~3.10"` will prevent the need for an update each time SPT gets a patch; this already breaks on 3.10.4.

Reply:
Change has been made, thanks for the spot 👍